src/configstore.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
Let's not mix the config store name with options. In the future we'll likely add more options so let's keep them separate.

Reply:
It's ok if I keep that file only to specify the settings as constants, and add a `configstore` prop to the `config.ts` file?

Reply:
Good idea 👍